spec/views/pages/edit.html.erb_spec.rb

```diff
@@ -5,7 +5,7 @@
 
   before do
     # Initialize models
-    page = Page.new(id: 1, question_text:, form_id: 1, answer_type: "email", answer_settings: nil)
+    page = Page.new(id: 1, question_text:, form_id: 1, answer_type: "email", answer_settings: nil, page_heading: nil)
```

Comment:
I'm just taking a closer look at what exactly made this test start failing here, but the test passes with this change.

Reply:
What was the failure? Wondering if it's related to this change on Friday: https://github.com/alphagov/forms-runner/pull/482/files

Reply:
Oh wait, that's runner - ignore me!

Reply:
Ah I think I've understood it now, I read the error properly. It looks like underneath the feature flag chunk in `app/views/pages/_form.html.erb` there's a section that's checking the `page_heading`. That test was running with the flag off so never ran the `page_heading` check, but now it's on the page every time it needs to get stubbed out.

Reply:
Just needed some rubber 🦆!